### PR TITLE
Updating form section to track active question/section.

### PR DESCRIPTION
**Pull request by ahmedhamodi**
Subtasks completed:
- track active question/section
- dynamically loading in the questions inside of a section.
- set up the template functions for how to integrate with the schema
- update schema to include title, description, and created_by

### CODE REVIEW

**Review by GodGreg — APPROVED** [review]

**Comment by GodGreg** [comment]
@xinhaoz Review please. 

Also I know you'll catch the form ID in the form creation url. It is currently not being used to load anything. 

But we may use it later so that it sets the users program on load if the is a valid program ID is in the url

File: src/Components/FormCreation/CreateEditForm.js [hunk 1]
```diff
@@ -125,36 +119,29 @@ function CreateEditForm() {
...
+        form.sections.map((section, key) => (
+          <FormWrapper key={key}>
+            <FormSection
+              key={key + "_section"}
+              numSections={form.sections.length}
+              sectionNum={key + 1}
+              sectionData={section}
+            />
+            {section.questions.map((_question, questionKey) => (
+              <FormCard
```
Comment (xinhaoz) [inline src/Components/FormCreation/CreateEditForm.js:107]: I think we should be rendering questions as a child of FormSection component

File: src/Components/FormCreation/CreateEditForm.js [hunk 2]
```diff
@@ -125,36 +119,29 @@ function CreateEditForm() {
...
+                key={questionKey + "_question"}
+                active={activeSection === key && activeQuestion === questionKey}
```
Comment (xinhaoz) [inline src/Components/FormCreation/CreateEditForm.js:109]: Once we render the questions inside FormSection, we can move the active question logic to form section as well and set the active question by whether or not the current section is active.

**Review by xinhaoz — CHANGES_REQUESTED** [review]

**Review by GodGreg — APPROVED** [review]

**Review by xinhaoz — APPROVED** [review]
LGTM!! Let's remember to change formId to programId

**Pull request merged**